### ISSUE TITLE
SSI_Poles orders issue

### ISSUE THREAD

**Issue by Ayubirad** [dagghe/pyOMA2#11]
Dear @dagghe 

When I changed the ordmin, ordmax, and step to 2, 50, 2, or any similar patterns, the following error occurs:

the SSI_funct.py, the SSI_Poles :

```
Fn[: len(fn), ii] = fn  # save the frequencies

IndexError: index 51 is out of bounds for axis 1 with size 51
```

Please help me solve it

**Comment by dagghe** [comment]
Dear @Ayubirad
Thank you for your feedback. I’m looking into the issue.

**Comment by dagghe** [comment]
Sorry for the delay, but I have been abroad. 
I have found the bug and will push a new release later today.

**Comment by Ayubirad** [comment]
Thanks @dagghe, please consider including the **MPC** and **MPD** criteria as well.

**Comment by dagghe** [comment]
fixed with release v0.5.1
